packages/cyberstorm/src/components/Layout/PackageSearchLayout/PackageSearchLayout.tsx

```diff
@@ -47,11 +47,17 @@ export class Filters {
   setKeywords: React.Dispatch<React.SetStateAction<string[]>>;
   availableCategories: CategoriesProps;
   setAvailableCategories: React.Dispatch<React.SetStateAction<CategoriesProps>>;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  pageObjectCount: number;
```

Comment:
~Why not just `pageCount`?~
Edit: oh wait this has nothing to do with pages, which makes the naming even more confusing. Why not use `totalCount` to match the consuming component's naming convention?